help-docs/marketplace/apps/give.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 13
+title: Give by SKALE
+id: give
+sidebar_label: Give
+---
+
+import img1 from '@site/static/img/tutorial/marketplace/poap.png';
+
+## Overview
+
+Send and receive tokens on the SKALE blockchain.
+
+### About this app
+
+Give allows community members to give and receive tokens onchain on the SKALE blockchain. Tokens are sent between members' [gmPASS-controlled smart accounts](/help-docs/key-features/gm-pass), creating a personal web of onchain connections between community members.

```

Comment:
Tokens are sent between members' custodial wallets

Reply:
@raymondfeng please clarify here

Reply:
I commented out the whole section. can add back later if desired